packages/components/select/src/use-select.ts

```diff
@@ -630,6 +660,22 @@ export function useSelect<T extends object>(originalProps: UseSelectProps<T>) {
     [slots, spinnerRef, spinnerProps, classNames?.spinner],
   );
 
+  const getClearButtonProps: PropGetter = useCallback(
+    (props = {}) => {
+      return {
+        ...props,
+        role: "button",
+        tabIndex: 0,
```

Comment:
![image](https://github.com/user-attachments/assets/46892dc1-98b4-4962-ad6d-1ec86f073e63)

in 2.5.0, clear button won't be focusable.

Reply:
@wingkwong so should I remove the `focusable` property from clear button?

Reply:
check the latest input implementation. `tabIndex` should be `-1`.